src/web/browserUtils.ts

```diff
@@ -1,4 +1,5 @@
 const isFirefox = navigator.userAgent.toLowerCase().includes('firefox');
 const isChromium = 'chrome' in window;
+const isMobile = /Android|webOS|iPhone|iPad|iPod|BlackBerry|BB|PlayBook|IEMobile|Windows Phone|Silk|Opera Mini/i.test(navigator.userAgent);
```

Comment:
How did you come up with this line? Is this copied from somewhere (from existing codebase) or newly added code snippet?

Reply:
I copied it from Expensify App. https://github.com/Expensify/App/blob/90dee7accae79c49debf30354c160cab6c52c423/src/libs/Browser/index.website.ts#L41

Reply:
Good, can we please add a comment in the code with this URL where it was copied from?

Reply:
Thanks